ocs_ci/deployment/helpers/hypershift_base.py

```diff
@@ -62,6 +65,35 @@ def wrapper(self, *args, **kwargs):
     return wrapper
 
 
+def get_random_hosted_cluster_name():
+    """
+    Get a random cluster name
+
+    Returns:
+        str: random cluster name
+    """
+    # getting the cluster name from the env data, for instance "ibm_cloud_baremetal3; mandatory conf field"
+    bm_name = config.ENV_DATA.get("baremetal").get("env_name")
+    ocp_version = get_latest_release_version()
+    hcp_version = "".join([c for c in ocp_version if c.isdigit()][:3])
+    match = re.search(r"\d+$", bm_name)
+    if match:
+        random_letters = "".join(
+            random.choice(string.ascii_lowercase) for _ in range(3)
+        )
+        cluster_name = (
+            "hcp"
+            + hcp_version
+            + "-bm"
+            + bm_name[match.start() :]
+            + "-"
+            + random_letters
+        )
```

Comment:
can we save this as a constant, that will get the random letters?

Reply:
Tbh not very clear for me. These are random letters attached to name to prevent creating 2 clusters with the same name.
* Why get_random_hosted_cluster_name is not a fixture  - it is used only for dynamically add hosted clusters, too specific task
* Why we do not have a function making this name with random letters in some common utility module - The name is very specific, so it makes no sense to reuse it for other resources

The function produces name similar to [hcp416-bm1-wtp](https://console-openshift-console.apps.ibm-baremetal1.qe.rh-ocs.com/multicloud/infrastructure/clusters/details/clusters/hcp416-bm1-wtp)